docs/tutor.rst

```diff
@@ -117,10 +117,17 @@ Contributing to Tutor
 Contributions to Tutor and its plugins are highly encouraged. Please adhere to the following guidelines:
 
 - **General Discussion**: Before addressing anything other than clear-cut bugs, start a discussion on the `official Open edX forum <https://discuss.openedx.org>`__. This facilitates reaching a consensus on a high-level solution.
-- **Pull Requests**: For changes to Tutor core or plugin-specific modifications, open a pull request on the `Tutor repository <https://github.com/overhangio/tutor/pulls>`__ or the corresponding plugin repository.
+- **Pull Requests**: For changes to Tutor core or plugin-specific modifications, open a pull request on the `Tutor repository <https://github.com/overhangio/tutor/pulls>`__ or the corresponding plugin repository. Take care to target your pull request to the proper branch:
+
+  - Target ``release`` if your change is compatible with the latest official Open edX release and it carries no major backwards-incompatibility nor risk of regression. This ensures that the latest stable release of Tutor benefits from bug fixes and incremental improvements. Once merged, your change will automatically be forward-ported to ``main``.
+  - Target ``main`` if your change is only compatible with Open edX's master branches and/or your change would be disruptive to production Tutor site operators. If merged, your change will become part of the next pending release branch (described below) and then incorporated into ``release`` at the time of the next named Open edX release.
+  - At the beginning each Open edX named release testing period, we split off from ``main`` a special pending release branch (e.g., ``redwood`` or ``sumac``). If your Tutor change is necessary for that pending release, merge it to said branch (it will be automatically forward-ported to ``main``). At the end of the testing period, the pending branch will be merged into ``release`` and then deleted.
```

Comment:
```suggestion
  - At the beginning of each Open edX named release testing period, we split off from ``main`` a special pending release branch (e.g., ``redwood`` or ``sumac``). If your Tutor change is necessary for that pending release, merge it to said branch. At the end of the testing period, the pending branch will be merged into ``release`` and then deleted.
```

Reply:
I removed the ` (it will be automatically forward-ported to ``main``)` part because it does not happen for release-specific branches (like sumac). That flow is only done for the changes merged to release.

Reply:
OK, so once the pending branch is merged into release, the changes will *then* be forward-ported to main, correct?

Reply:
Yes, that is correct.